index.js

```diff
@@ -94,9 +94,10 @@ async function underPressure (fastify, opts) {
         response: {
           200: {
             type: 'object',
-            properties: {
-              status: { type: 'string' }
-            }
+            properties: Object.assign(
+              { status: { type: 'string' } },
+              opts.exposeStatusRoute.routeResponseSchemaOpts
```

Comment:
question: is there a chance that `opts.esposeStatusRoute` is null or undefined and hence that this will throw?

Reply:
yes, it can be. It's already taken care of though. This code is inside `if (opts.exposeStatusRoute) { ...` (10 lines above, hidden by github 😁 )